tests/test_schema.py

```diff
@@ -1766,25 +1778,27 @@ def test_schema():
         """,
 """
 <program>
-  <functioncall>
-    <function>
-      <funcexpr>
-        <identifier/>
-        <parameters/>
-        <body>
-          <functioncall>
-            <function>
-              <identifier name="alert"/>
-            </function>
-            <arguments>
-              <string>foo</string>
-            </arguments>
-          </functioncall>
-        </body>
-      </funcexpr>
-    </function>
-    <arguments/>
-  </functioncall>
+  <groupingoperator>
```

Comment:
Do the changes in this file mean that this PR is backward-incompatible? I'm all in for it, what I mean is, does this grant a major version bump?

Reply:
It is definitely backward-incompatible. And yes, it probably makes sense to go for a major version bump.